grafana/risingwave-dashboard.py

```diff
@@ -99,6 +99,11 @@ def timeseries_bytes_per_sec(self, title, targets):
         return TimeSeries(title=title, targets=targets, gridPos=gridPos, unit="Bps", fillOpacity=10,
                           legendDisplayMode="table", legendPlacement="right", legendCalcs=["max"])
 
+    def timeseries_requests_per_sec(self, title, targets):
```

Comment:
It looks like `timeseries_ops`, can we use `timeseries_ops` directly?

Reply:
This metric is intended for the high-level read and write, not for each specific operation (e.g. put, get, head, etc.)